### ISSUE TITLE
train mAP@0.5 is low

### ISSUE THREAD

**Issue by KevenLee** [ultralytics/yolov5#777]
## ❔Question
I using 72 Images to train yolov5s model, but mAP@0.5 is so low, something may be wrong with it.

## Additional context
![image](https://user-images.githubusercontent.com/10877402/90501706-53212a00-e17f-11ea-9ce7-6d37bbab9cd6.png)

traing 
python train.py --img-size 2560 --batch-size 8 --epochs 30 --data ./data/coco.yaml --cfg ./models/yolov5s.yaml --weights ./yolov5s.pt

coco.yaml
train: yolov5/dataSet/images/train/  # 118287 images
val: yolov5/dataSet/images/train/  # 5000 images

# number of classes
nc: 6

# class names
names: ['effusion', 'leakage flange', 'leak', 'cold air leakage', 'dust', 'flame']

yolov5s.yaml
# parameters
nc: 6  # number of classes

Data link:
link: https://pan.baidu.com/s/1ErS_eV614q7gOrjdByVzbw 
code: fr41

How should I do.

**Comment by glenn-jocher** [comment]
@KevenLee recommend you start from the tutorials, and use default settings rather than making up your own.

You may also want to consider waiting for training to finish rather than complaining about your results at epoch 5.

**Comment by KevenLee** [comment]
> @KevenLee recommend you start from the tutorials, and use default settings rather than making up your own.
> 
> You may also want to consider waiting for training to finish rather than complaining about your results at epoch 5.

I start yolov5 from the tutorials, and using default settings except # number of classes. And I found that using coco128 dataset, mAP@.5 looks normal. But using my dataset seems wrong.  And train_batch1.jpg looks normal, just as follow:
![train_batch1](https://user-images.githubusercontent.com/10877402/90602033-a781e380-e22b-11ea-9235-aa4e79c64351.jpg)

what should I do now.

**Comment by KevenLee** [comment]
> @KevenLee recommend you start from the tutorials, and use default settings rather than making up your own.
> 
> You may also want to consider waiting for training to finish rather than complaining about your results at epoch 5.

I start yolov5 from the tutorials, and using default settings except # number of classes. And I found that using coco128 dataset, mAP@.5 looks normal. But using my dataset seems wrong.  And train_batch1.jpg looks normal, just as follow:
![train_batch1](https://user-images.githubusercontent.com/10877402/90602033-a781e380-e22b-11ea-9235-aa4e79c64351.jpg)

what should I do now.

**Comment by glenn-jocher** [comment]
@KevenLee your train_batch1.jpg does not look normal at all, they look mostly random to me. If a human can not be expected to learn your labels, ML won't either.

**Comment by Ownmarc** [comment]
Yea, neural nets wont learn anything with 72 images of this kind! Gather more images (probably x50) and try to have consistent annotations (some look tight while others are very large)!

**Comment by KevenLee** [comment]
> @KevenLee your train_batch1.jpg does not look normal at all, they look mostly random to me. If a human can not be expected to learn your labels, ML won't either.

Thanks for your reply. It looks abnormal, but, actually normal. We aim to detect leakage, effusion, gas leak in the factory. The degree of leakages are different, so looks random and abnormal. The resolutions are large of the data, and scale of the targets varies greatly. So, I think the data augmentation of yolov5 may not be suitable for my dataset. Otherwise, what should I do for this data

**Comment by KevenLee** [comment]
> Yea, neural nets wont learn anything with 72 images of this kind! Gather more images (probably x50) and try to have consistent annotations (some look tight while others are very large)!

I think, it should overfit the training set with very small amount of data

**Comment by Frank1126lin** [comment]
@KevenLee I believe you need just label the leakage class and leave the nomal valve. Thus the valve has different type and shape, but the leak air or oil has similar pattern. Maybe you could try in this way . And try to use more imgs.

**Comment by KevenLee** [comment]
> @KevenLee I believe you need just label the leakage class and leave the nomal valve. Thus the valve has different type and shape, but the leak air or oil has similar pattern. Maybe you could try in this way . And try to use more imgs.

Thanks for your suggest.
